tests/test_writer.py

```diff
@@ -1949,6 +1949,7 @@ def _get_parent_bookmark(current_indent, history_indent, bookmarks) -> Any:
         bookmarks.append(new_bookmark)
 
 
+pytest.importorskip("fpdf")
```

Comment:
That is something I'm very uncertain about:

1. I think it needs to be within the test. If you write it like this, I think the whole module is skipped if the import fails (or the statement doesn't do anything)
2. I'm not sure if using this is a good idea. fpdf is a very lightweight dependency and shouldn't be a problem to install it.